packages/avatars/demo/~patterns/stories/MenuStory.tsx

```diff
@@ -5,93 +5,80 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import React, { useState } from 'react';
-import { Story } from '@storybook/react';
-import Icon from '@zendeskgarden/svg-icons/src/16/chevron-down-stroke.svg';
-import { PALETTE } from '@zendeskgarden/react-theming';
+import React, { useCallback, useState } from 'react';
+import { StoryFn } from '@storybook/react';
+import { getColor } from '@zendeskgarden/react-theming';
 import { Grid } from '@zendeskgarden/react-grid';
-import {
-  Dropdown,
-  Trigger,
-  Menu,
-  MediaItem,
-  MediaFigure,
-  MediaBody,
-  ItemMeta
-} from '@zendeskgarden/react-dropdowns.legacy';
-import { Button } from '@zendeskgarden/react-buttons';
-import { Avatar } from '@zendeskgarden/react-avatars';
+import { Menu, Item } from '@zendeskgarden/react-dropdowns';
+import { Avatar, IAvatarProps } from '@zendeskgarden/react-avatars';
+import { useTheme } from 'styled-components';
 
-export const MenuStory: Story = ({ isCompact }) => {
-  const [highlightedItem, setHighlightedItem] = useState<number | null>();
-  const [isOpen, setIsOpen] = useState<boolean | undefined>();
+const items: {
+  value: string;
+  label: string;
+  avatarProps: IAvatarProps;
+}[] = [
+  {
+    value: 'linden',
+    label: 'Linden',
+    avatarProps: {
+      status: 'away',
+      badge: undefined
+    }
+  },
+  {
+    value: 'reed',
+    label: 'Reed',
+    avatarProps: {
+      status: 'available',
+      badge: undefined
+    }
+  },
+  {
+    value: 'sage',
+    label: 'Sage',
+    avatarProps: {
+      status: undefined,
+      badge: '3'
+    }
+  }
+];
+
+export const MenuStory: StoryFn = ({ isCompact }) => {
+  const [highlightedValue, setHighlightedValue] = useState<string | null>();
+  const theme = useTheme();
+  const surfaceColor = getColor({ variable: 'background.raised', theme });
+  const highlightedColor = getColor({ variable: 'background.primary', theme });
```

Comment:
This doesn't appear to be correct for dark mode as the surface doesn't match the backing focus color.

Reply:
I used this color because any color with opacity won't look correct as a `surfaceColor` (it will just look like no surface color is applied). Attaching a screenshot of how the color looks if I match the focused item background exactly. `background.primary` was the closest color given the limitaiton. Unless you see another option that I missed?

PS. Ignore the spacing bug, fix is WIP.

<img width="87" alt="Screenshot 2024-07-12 at 11 48 20 AM" src="https://github.com/user-attachments/assets/0cec9030-8c33-406d-a66a-5efede358879">

Reply:
I used this color because any color with opacity won't look correct as a `surfaceColor` (it will just look like no surface color is applied). Attaching a screenshot of how the color looks if I match the focused item background exactly. `background.primary` was the closest color given the limitaiton. Unless you see another option that I missed?

PS. Ignore the spacing bug, fix is WIP.

<img width="87" alt="Screenshot 2024-07-12 at 11 48 20 AM" src="https://github.com/user-attachments/assets/0cec9030-8c33-406d-a66a-5efede358879">

Reply:
I used this color because any color with opacity won't look correct as a `surfaceColor` (it will just look like no surface color is applied). Attaching a screenshot of how the color looks if I match the focused item background exactly. `background.primary` was the closest color given the limitaiton. Unless you see another option that I missed?

PS. Ignore the spacing bug, fix is WIP.

<img width="87" alt="Screenshot 2024-07-12 at 11 48 20 AM" src="https://github.com/user-attachments/assets/0cec9030-8c33-406d-a66a-5efede358879">

<img width="96" alt="Screenshot 2024-07-12 at 11 51 11 AM" src="https://github.com/user-attachments/assets/0424ed25-c96c-4407-8aaf-9569fed3d023">

Reply:
Ok, it's good to know that this is the best we can do under the circumstances.